map/local_ads_supported_types.cpp

```diff
@@ -0,0 +1,163 @@
+#include "map/local_ads_manager.hpp"
+
+#include <initializer_list>
+
+void LocalAdsManager::FillSupportedTypes()
+{
+  m_supportedTypes.Append<std::initializer_list<std::initializer_list<std::string>>>(
+      {{"amenity", "atm"},
```

Comment:
Is it possible to get rid of this duplication? amenity, amenity, amenity, amenity....